### ISSUE TITLE
Repository name

### ISSUE THREAD

**Issue by mackensen** [Swarthmore/moodle-local_accessibility_filescan#61]
The name of the repository implies that the plugin should be installed in local/accessibility_filescan , but it actually needs to be installed in local/a11y_check. That misunderstanding led to #60.

**Comment by mackensen** [comment]
Also not a bug; but the state of master is confusing.
